src/main/java/Duke/TaskList.java

```diff
@@ -0,0 +1,91 @@
+package duke;
+
+import java.util.ArrayList;
+
+/**
+ * TaskList object class that stores ArrayList of Tasks.
+ */
+public class TaskList {
+
+    private ArrayList<Task> listOfTasks;

```

Comment:
Suggest to name this as tasks instead, just to keep things short and simple
```suggestion
    private ArrayList<Task> tasks;

```

Reply:
Thanks!